impl/src/main/java/org/ehcache/impl/internal/statistics/DefaultTierStatistics.java

```diff
@@ -223,21 +217,13 @@ public long getMisses() {
   public long getPuts() {
     return put.sum(EnumSet.of(StoreOperationOutcomes.PutOutcome.PUT)) +
            putIfAbsent.sum(EnumSet.of(StoreOperationOutcomes.PutIfAbsentOutcome.PUT)) +
-           put.sum(EnumSet.of(StoreOperationOutcomes.PutOutcome.REPLACED)) +
            compute.sum(EnumSet.of(StoreOperationOutcomes.ComputeOutcome.PUT)) +
            computeIfAbsent.sum(EnumSet.of(StoreOperationOutcomes.ComputeIfAbsentOutcome.PUT)) +
            replace.sum(EnumSet.of(StoreOperationOutcomes.ReplaceOutcome.REPLACED)) +
            conditionalReplace.sum(EnumSet.of(StoreOperationOutcomes.ConditionalReplaceOutcome.REPLACED)) -
            compensatingCounters.puts;
   }
 
-  public long getUpdates() {
-    return put.sum(EnumSet.of(StoreOperationOutcomes.PutOutcome.REPLACED)) +
```

Comment:
Can't we still count the `replace`? Or they are now puts?

Reply:
We no longer distinguish between puts that add new entries and puts that replace old ones - they're all just puts now.

Reply:
Ok